go.mod

```diff
@@ -4,7 +4,8 @@ go 1.13
 
 require (
 	github.com/JankariTech/GoBikramSambat v0.0.0-20200507040640-7d5f1ba6f8da
-	github.com/cucumber/godog v0.9.0
+	github.com/cucumber/godog v0.12.6
```

Comment:
Since you upgraded the godog version to `0.12.6`, it requires a go runtime `1.16 or greater`. So, It is better if we upgrade the go version in `go.mod` file to the latest one.

Reply:
I have made a commit, which resolves this.